src/core/classes/contract-deployer.ts

```diff
@@ -162,6 +252,22 @@ export class ContractDeployer extends RPCConnectionHandler {
 
   /**
    * Deploys a new Split contract
+   *
+   * @remarks Deploys a Split contract and returns the address of the deployed contract
+   *
+   * @example
+   * ```javascript
+   * const contractAddress = await sdk.deployer.deploySplit({
+   *   name: "My Split",
+   *   primary_sale_recipient: "your-address",
+   *   recipients: [
+   *    {
+   *      address: "your-address",
+   *      sharesBps: 100 * 100, // 100%
```

Comment:
Would be nice to show a 2 person split here like a 80/20

Reply:
added